Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -92,6 +109,32 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
         self.present(alertController, animated: true, completion: nil)
     }
     
+    // MARK: - Private instance methods
+
+    private func toggleView() {
```

Comment:
A probably better function name is `checkSyncStatus`. `toggleView` sounds like a simple hide-if-visible-and-show-if-hidden sort of operation.